projects/igniteui-angular/src/lib/toast/toast.component.ts

```diff
@@ -300,10 +345,9 @@ export class IgxToastComponent implements IToggleView, OnInit, OnDestroy {
      */
     public hide(): void {
         this.onHiding.emit(this);
-        this.isVisible = false;
+        this._isVisible = false;
         this.onHidden.emit(this);
```

Comment:
same as above, this will fire too early because of the animation.

Reply:
And here.